cterasdk/edge/network.py

```diff
@@ -176,3 +177,57 @@ def iperf(self, address, port=5201, threads=1, protocol=IPProtocol.TCP, directio
             return task.result.res
         except TaskError as error:
             return error.task.result.res
+
+    def get_static_routes(self):
+        """
+        Get all Static Routes
+        """
+        return self._gateway.get('/config/network/static_routes')
+
+    def set_static_route(self, source_ip, destination_ip, destination_mask):
+        """
+        Set a Static Route
+
+        :param str source_ip: The source Filer IP (Gateway IP)
+        :param str destination_ip: The destination IP
+        :param str destination_mask: The destination IP subnet mask CIDR Notation (1-32)
+        """
+        param = Object()
+        param.GwIP = source_ip
+        param.DestIpMask = f'{destination_ip}_{destination_mask}'
+        try:
+            res = self._gateway.add('/config/network/static_routes', param)
+            logging.getLogger().info(
+                "Static Route updated. %s", {'Source': param.GwIP, 'Destination': param.DestIpMask})
+            return res
+        except CTERAException as error:
+            logging.getLogger().error("Static route creation failed.")
+            raise CTERAException('Static route creation failed', error)
+
+    def del_static_route(self, destination_ip, destination_subnet_mask):
+        """
+        Delete a Static Route
+
+        :param str destination_ip: The destination IP
+        :param str destination_subnet_mask: The destination IP subnet mask CIDR Notation (1-32)
+        """
+        try:
+            response = self._gateway.delete(f'/config/network/static_routes/{destination_ip}_{destination_subnet_mask}')
+            logging.getLogger().info(
+                "Static Route deleted. %s", {'Destination': destination_ip + '_' + destination_subnet_mask})
+            return response
+        except CTERAException as error:
+            logging.getLogger().error("Static route deletion failed.")
+            raise CTERAException('Static route deletion failed', error)
+
+    def clean_all_static_routes(self):
+        """
+        Clean all Static Routes
+        """
+        try:
+            self._gateway.execute('/config/network', 'cleanStaticRoutes')
+            logging.getLogger().info('All Static Routes cleaned Successfully')
+            return 'All Static Routes cleaned Successfully'
```

Comment:
No need to return strings. If no errors occurred, then the command should return nothing